### DISCUSSION TITLE
problem with # (hash) anywhere for tagging

### DISCUSSION THREAD

**Discussion by muralikodali** [srid/neuron#459] [General]
putting hash tag any where in the notes for tagging is a convenient thing. but, there is a headache with it. There are places where we have to use # (hash) other than tagging. It is giving very weird tags in search like #RRGGBB (color code), #DIV, #- etc. At most we can give three to four tags for a note. It is not a burden to give those tags on a single line under the date header. I request you to consider this problem and change the way of tagging.

**Comment by srid** [comment]
You can escape them using backslash, eg: `\#DIV`.  Also, hash is ignored in code blocks.

Once neuron gets [plugin-based architecture](https://github.com/srid/neuron/discussions/462), we can make inline tags a plugin and make it possible to be enabled or disabled by the user.

**Comment by srid** [comment]
https://github.com/srid/neuron/issues/519
